main/lsp/LSPConfiguration.cc

```diff
@@ -162,7 +162,9 @@ string urlDecode(string_view uri) {
         auto from = uri.substr(pos, 3);
         // add replacement only if % is actually followed by exactly 2 hex digits
         if (from.size() == 3 && isxdigit(from[1]) && isxdigit(from[2])) {
-            auto to = absl::HexStringToBytes(from.substr(1));
+            string to;
+            auto valid = absl::HexStringToBytes(from.substr(1), &to);
+            ENFORCE(valid, "We checked it was valid above");
```

Comment:
They deprecated the version of the function we were using, which got converted to an error with `-Werror`.